content/tutorials/numpy_integration/grass_numpy_integration.qmd

```diff
@@ -0,0 +1,253 @@
+---
+title: "Using GRASS, NumPy, and Landlab for Scientific Modeling"
+author: "Anna Petrasova"
+date: 2025-10-01
+date-modified: today
+categories: [NumPy, raster, terrain, intermediate, Landlab, Python, hydrology, matplotlib]
+description: >
+  Learn how to integrate NumPy arrays with GRASS tools
+  on an example using Landlab modeling framework.
+image: thumbnail.webp
+other-links:
+- text: "Check out new xarray-grass interface"
+  href: "https://github.com/lrntct/xarray-grass"
+  icon: link
```

Comment:
Interesting feature. Perhaps the place to put link to the doc? No need to overdo it, but since there is an external related resource, something really on-topic would make sense. One of?

https://grass.osgeo.org/grass-devel/manuals/python_intro.html
https://grass.osgeo.org/grass-devel/manuals/libpython/grass.tools.html
https://grass.osgeo.org/grass-devel/manuals/libpython/grass.tools.html#grass.tools.session_tools.Tools